index.js

```diff
@@ -32,11 +26,7 @@ module.exports = function S3Router(options) {
     s3Options.endpoint = options.endpoint;
   }
 
-  const s3 = new AWS.S3(Object.assign({
```

Comment:
Let me know if I'm breaking some use case here. I think this is all you need to do.